util/common/src/main/java/module-info.yml

```diff
@@ -0,0 +1,21 @@
+name: org.hibernate.search.util.common
+
+requires:
+  - module: org.jboss.jandex
+  - module: org.jboss.logging
+
+exports:
+  - package: org\.hibernate\.search\.util\.common\b(\..+)*\.impl
+    pattern: true
```

Comment:
This also is a new option; without it we'd need to list all those `impl` packages and add the same `to`s to each